inc/modules/export/class-exporthelpers.php

```diff
@@ -28,15 +28,16 @@ public function mapBookDataAndContent( $post_data, $metadata, $post_number, $opt
 		$needs_tidy_html = $options['needs_tidy_html'] ?? false;
 		$endnotes = $options['endnotes'] ?? false;
 		$footnotes = $options['footnotes'] ?? false;
+		$slug_as_href = $options['slug_as_href'] ?? false;
 
 		$data = [
 			'id' => $post_data['ID'],
 		];
-		$data['post_type_class'] = str_replace( '_','-', $post_type_identifier ); // This class is used to map with the SCSS class in buckram Ex: front-matter
+		$data['post_type_class'] = str_replace( '_', '-', $post_type_identifier ); // This class is used to map with the SCSS class in buckram Ex: front-matter
 		$method = studly_case( $post_type_identifier );
 		$taxonomy_method = "get{$method}Type";
 		$data['subclass'] = $this->taxonomy->{$taxonomy_method}( $post_data['ID'] );
-		$data['slug'] = "{$data['post_type_class']}-{$post_data['post_name']}";
+		$data['slug'] = $slug_as_href ? $post_data['post_name'] : "{$data['post_type_class']}-{$post_data['post_name']}";
```

Comment:
Checks to see how to handle `slugs` since we use them for referencing internal links in EPUBs